components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -226,6 +226,28 @@ const PrepopulatedEngine brave_search = {
     PREPOPULATED_ENGINE_ID_BRAVE,
 };
 
+const PrepopulatedEngine brave_bing = {
```

Comment:
Most of values are copied from https://source.chromium.org/chromium/chromium/src/+/main:components/search_engines/prepopulated_engines.json;l=62
```
    "bing": {
      "name": "Bing",
      "keyword": "bing.com",
      "favicon_url": "https://www.bing.com/sa/simg/bing_p_rr_teal_min.ico",
      "search_url": "https://www.bing.com/search?q={searchTerms}&PC=U316&FORM=CHROMN",
      "suggest_url": "https://www.bing.com/osjson.aspx?query={searchTerms}&language={language}&PC=U316",
      "image_url": "https://www.bing.com/images/detail/search?iss=sbiupload&FORM=CHROMI#enterInsights",
      "new_tab_url": "https://www.bing.com/chrome/newtab",
      "image_url_post_params": "imageBin={google:imageThumbnailBase64}",
      "type": "SEARCH_ENGINE_BING",
      "id": 3
    },
```
Keyword is changed to `:b`.
`&PC=U316&FORM=CHROMN` is removed from `search_url`
`FORM=CHROMI` is removed from `image_url`.

Reply:
I think you can use `ModifyEngineParams` which I defined above to create a new type 😄 This way, you only pass in what is different and we retain the other values from upstream

Reply:
@bsclifton Thanks! and changing `image_url` seems fine? - Removing `FORM=CHROMI` from `image_url`

Reply:
Updated. We can check what is updated more easily :)